code/go/0chain.net/blobbercore/convert/response_creator.go

```diff
@@ -2,9 +2,9 @@ package convert
 
 import (
 	"encoding/json"
+	blobbergrpc "github.com/0chain/blobber/code/go/0chain.net/blobbercore/blobbergrpc/proto"
```

Comment:
The old code naming is not good. should we name it like go-lint rule? for example
```
package blobber 

type BlobberGrpc struct

```

the `go-lint` will ask we rename `BlobberGrpc` with `Grpc`. Because the package name already has `blobber` now. It works better with `blobber.Grpc` than `blobber.BlobberGrpc`

Reply:
Golint is deprecated long back, these are stylechecks which already have been imbued into the CI with golangci-lint. The ci isnt complaining which means it should be good to go, because changing the package name will require changes again in the complete PR and seeing the current PR has >6000 lines of change, I suggest we keep that issue for later.

Reply:
. leave them there for now. Please keep an eyes on your mind in new code file. thanks.